src/plugins/dashboard/kibana.jsonc

```diff
@@ -34,7 +34,8 @@
       "usageCollection",
       "taskManager",
       "serverless",
-      "noDataPage"
+      "noDataPage",
+      "security"
```

Comment:
Not a blocker for this PR, but would you mind listing the APIs that you add to use from the security plugin there? The idea behind security-in-core is to allow plugins to access user information from Core directly, without depending on the security plugin explicitly, so we'll probably want to work on exposing those from Core.